blog-article.html

```diff
@@ -0,0 +1,454 @@
+                                    Our blog 
+                                    <span class="sidebar__num">
+                                        (12)
+                                    </span>
+                                </a>
+                            </li>
+                            <li class="sidebar__item">
+                                <a href="#" class="sidebar__link">
+                                    Metaverse 
+                                    <span class="sidebar__num">
+                                        (24)
+                                    </span>
+                                </a>
+                            </li>
+                            <li class="sidebar__item">
+                                <a href="#" class="sidebar__link">
+                                    Gaming 
+                                    <span class="sidebar__num">
+                                        (24)
+                                    </span>
+                                </a>
+                            </li>
+                            <li class="sidebar__item">
+                                <a href="#" class="sidebar__link">
+                                    Art
+                                    <span class="sidebar__num">
+                                        (24)
+                                    </span>
+                                </a>
+                            </li>
+                            <li class="sidebar__item">
+                                <a href="#" class="sidebar__link">
+                                    Robot 
+                                    <span class="sidebar__num">
+                                        (24)
+                                    </span>
+                                </a>
+                            </li>
+                            <li class="sidebar__item">
+                                <a href="#" class="sidebar__link">
+                                    Lauchpad 
+                                    <span class="sidebar__num">
+                                        (24)
+                                    </span>
+                                </a>
+                            </li>
+                            <li class="sidebar__item">
+                                <a href="#" class="sidebar__link">
+                                    Crypto 
+                                    <span class="sidebar__num">
+                                        (430)
+                                    </span>
+                                </a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+                <div class="aside-box">
+                    <strong class="aside__title">
+                        Tag
+                    </strong>
+                    <ul class="tag">
```

Comment:
How about rename it from `tag` to `tags`? Because list its a log of tags, not one

Reply:
Done